Library/Homebrew/cmd/orphaned.rb

```diff
@@ -0,0 +1,13 @@
+#:  * `orphaned`:
+#:    Lists kegs that were installed as dependencies of other kegs, but are no
```

Comment:
Let's use `installed formulae` rather than `kegs`; it's widely misunderstood as a term.